models/Import/Service/SendCalculatedResultService.php

```diff
@@ -59,7 +59,12 @@ public function sendByDeliveryExecutionId(string $deliveryExecutionId): array
         [$scoreTotal, $scoreTotalMax] = $this->getScores($outcomeVariables);
 
         $isFullyGraded = $this->checkIsFullyGraded($deliveryExecutionId, $outcomeVariables);
-        $gradingTimestamp = time();
+
+        $gradingTimestamp = null;
+        if ($isFullyGraded) {
```

Comment:
It should be sent in case if results were updated, the value should be the latest update timestamp

Reply:
Changed it according to our slack conversation.